src/stave.ts

```diff
@@ -308,8 +308,8 @@ export class Stave extends Element {
   }
 
   // Volta functions
-  setVoltaType(type: number, number_t: string, y: number): this {
-    this.modifiers.push(new Volta(type, number_t, this.x, y));
+  setVoltaType(type: number, number: string, y: number): this {
+    this.modifiers.push(new Volta(type, number, this.x, y));
```

Comment:
I agree, but this is actually a deeper issue, since Volta has a property named `number: string`... so I was just trying to follow the naming convention of the class itself. Should we fix Volta as well? What should we call that field? 

See: https://github.com/0xfe/vexflow/blob/2bf918a09b59d0d230511d076fe69832e0c3f3fa/src/stavevolta.ts#L32-L45

Reply:
OK, I decided to rename this param to `label` since it is the label at the left corner of the volta bracket.

```
  setVoltaType(type: number, label: string, y: number): this {
```

In a separate PR, we should rename the field inside the Volta class. Yes, the label is usually a number followed by a period. But I agree it is weird to have a property called `number` that is actually a `string`.